tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -220,3 +221,203 @@ ZTEST_F(ascs_test_suite, test_abort_client_operation_if_callback_not_registered)
 		      param->response_code);
 	zassert_equal(0x00, param->reason, "unexpected Reason 0x%02x", param->reason);
 }
+
+ZTEST_F(ascs_test_suite, test_release_ase_on_acl_disconnection_client_terminates_cis)
+{
+	const struct bt_gatt_attr *ase;
+	struct bt_bap_stream *stream = &fixture->stream;
+	struct bt_conn *conn = &fixture->conn;
+	struct bt_iso_chan *chan;
+	uint8_t ase_id;
```

Comment:
Keep definitions from longest to shortest for code style conformity.
Applies in multiple places.

Reply:
done